index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="es" xml:lang="es">
+<head>
+  <title>Koombea - Training</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name = "keywords" content = "koombea, html, css, javascript" />
+  <meta name = "description" content = "Training how to be a better front end." />
```

Comment:
Same as previous line
```suggestion
  <meta name="description" content="Training how to be a better front end." />
```